go.mod

```diff
@@ -6,10 +6,10 @@ require (
 	github.com/disintegration/imaging v1.6.2
 	github.com/ericpauley/go-quantize v0.0.0-20200331213906-ae555eb2afa4
 	github.com/fogleman/gg v1.3.0
-	github.com/julienschmidt/httprouter v1.3.0
 	github.com/patrickmn/go-cache v2.1.0+incompatible
 	github.com/sirupsen/logrus v1.8.1
 	github.com/stretchr/testify v1.7.0
+	goji.io v2.0.2+incompatible
```

Comment:
Do we want to avoid the newer `v3.0.0` version of Goji? Development on the project looks paused for a few years, are you thinking to use the latest stable version because of that?

Reply:
I actually didn't notice this to be honest. This was the version that Go selected when I ran `go get github.com/goji/goji`

Reply:
I updated to v3 - nice job noticing that :D